.pre-commit-config.yaml

```diff
@@ -18,6 +18,12 @@ repos:
     language: system
     files: >-
       ^setup\.py$
+  - id: docs-rebuild
+    name: Checks that generated docs are in-sync with commit
+    entry: python -m ansiblelint.generate_docs
```

Comment:
This looks heavy. I'd move it further, after most of the linters, just before flake8.